src/compiler/compiler.ts

```diff
@@ -32,15 +32,15 @@ export function compile(spec, theme?) {
       padding: 'auto'
     },
     ['viewport', 'background'].reduce(function(topLevelConfig, property) {
-      const value = model.config(property);
+      const value = model.config()[property];
```

Comment:
should `model.config()` and `model.sceneConfig()` be cached in consts above?  

(No worry if you prefer this)